internal/server/infrastructure/mysql/repository/log_repository.go

```diff
@@ -58,3 +58,20 @@ func (r *LogRepository) CTRSave(ctx context.Context, ctrLog *domain.CTRLog) erro
 	})
 	return err
 }
+
+func (r *LogRepository) CTRList(ctx context.Context) ([]domain.CTRLog, error) {
```

Comment:
Please write the doc string! Refer to https://qiita.com/shibukawa/items/8c70fdd1972fad76a5ce#%E6%96%B0pkggodev%E3%81%AE%E5%A0%B4%E5%90%88